crates/bevy_ui/src/geometry.rs

```diff
@@ -132,22 +132,22 @@ pub struct UiRect {
     pub bottom: Val,
 }
 
-impl UiRect {
+impl Frame {
     pub const DEFAULT: Self = Self {
         left: Val::DEFAULT,
         right: Val::DEFAULT,
         top: Val::DEFAULT,
         bottom: Val::DEFAULT,
     };
```

Comment:
I think it makes sense to either have no default at all or set this to `0`.
Not sure if that should be changed in this PR though.

Reply:
Yeah, I'd prefer to swap that later.

Reply:
> I think it makes sense to either have no default at all or set this to `0`. Not sure if that should be changed in this PR though.

I think this is fixed in the remove `Val::Undefined` pr.